app/views/layouts/_navbar.html.erb

```diff
@@ -0,0 +1,31 @@
+<nav class="navbar navbar-expand-sm navbar-dark bg-dark">
+      <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarsExample03" aria-controls="navbarsExample03" aria-expanded="false" aria-label="Toggle navigation">
+        <span class="navbar-toggler-icon"></span>
+      </button>
+
+      <div class="collapse navbar-collapse" id="navbarsExample03">
+        <ul class="navbar-nav mr-auto">
+          <li class="nav-item active">
+            <%= link_to 'Home', user_notes_path, class: 'nav-link' %>
```

Comment:
after login is fine. the question is still outstanding.

`why Home is linked to notes?`

Reply:
fixed